rcgen/src/lib.rs

```diff
@@ -663,6 +662,77 @@ impl AsRef<[u8]> for SerialNumber {
 	}
 }
 
+mod oid {
```

Comment:
I wasn't aware either that there has already been an oid module. If it's in a separate file we can keep it there (and maybe have pub(crate) instead of pub).